parser/routing.go

```diff
@@ -0,0 +1,97 @@
+package parser
+
+import (
+	"github.com/deis/workflow-cli/cmd"
+	docopt "github.com/docopt/docopt-go"
+)
+
+// Routing displays all relevant commands for `deis routing`.
+func Routing(argv []string) error {
+	usage := `
+Valid commands for routing:
+
+routing:list       list routability for an app
```

Comment:
It's a bit confusing, true, but `list` is how we do other configuration based things. Maybe it should have no `list` and only show up in the `app:info` page? Not sure... UX thing